tests/unit/App.spec.ts

```diff
@@ -1092,3 +1096,111 @@ describe('Restoration - App page', () => {
     expect(wrapper.findComponent(Actions).exists()).toBe(true)
   })
 })
+
+describe('Breadcrumbs for firms - Without Easy Legal Name Fix', () => {
```

Comment:
I separated the following tests into two blocks, one with the easy legal name fix and one without.